src/net.h

```diff
@@ -629,6 +627,9 @@ class CNode
 class NetEventsInterface
 {
 public:
+    /** Mutex for anything that is only accessed via the msg processing thread */
+    static Mutex g_mutex_msgproc_thread;
```

Comment:
nit: since it is not required that the variables protected by this mutex should only be used from a single thread in future, maybe drop `_thread` from the name of this mutex.

nit: it seems like unwritten convention to put `_mutex` at the end: `foobar_mutex`, I think this is the first one to use `mutex_foobar`

Reply:
Renamed